OwnTube.tv/src/screens/HomeScreen/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { StyleSheet } from "react-native";
+
+export const styles = (theme: any) =>
```

Comment:
We are using TS so please forget about any in the PRs :)

Reply:
Please apply it in other places as well

Reply:
![image](https://github.com/OwnTube-tv/web-client/assets/112472401/604981c5-41ee-43c3-8f41-b483129da4bf)
I get this error when trying to run it without any

i think it has something to do with `tsconfig.js` and the `"compilerOptions": {"strict": true}`

Reply:
This is normal, we have to keep it but fix your issue using RN types